### ISSUE TITLE
Send file by email without user confirmation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#169]
```
Enhancement : 
As the camera command show the send of a file is possible, what about add a few 
command to get back files. 
=> CD/LIST/PWD/GET should be fine.

If someone stole my phone, it woul'd be usefull if I can get my data back or to 
get the photo if the thief has take photo of himself. 



```

Original issue reported on code.google.com by `esteve.g...@gmail.com` on 17 May 2011 at 12:26

**Comment by GoogleCodeExporter** [comment]
```
It's already possible with the send command by xmpp.
I'll change your bug to have files by mail but with the actual implementation 
we need to confirm the email on the phone.

For the other commands you can use the "cmd" command that allow all shell 
commands ! 
```

Original comment by `Florent....@gmail.com` on 17 May 2011 at 1:07
- Changed title: **Send file by email without user confirmation**
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 17 May 2011 at 1:07
- Changed title: **[ER] Send file by email without user confirmation**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 31 Aug 2011 at 10:55
- Changed title: **Send file by email without user confirmation**

**Comment by GoogleCodeExporter** [comment]
```
It's not so easy to program. Everyone is welcomed to contribute.
```

Original comment by `fschm...@gmail.com` on 30 Nov 2011 at 10:16
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision e7f28cff8529.
```

Original comment by `yako...@gmail.com` on 7 Mar 2015 at 11:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in beta available on market 5.0.1.84
```

Original comment by `Florent....@gmail.com` on 8 Mar 2015 at 6:07
- Changed state: **FixedInBeta**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 1 May 2015 at 5:42
- Changed state: **Deployed**
